--- parser.cpp	2015-09-29 12:10:54.081924830 +0300
+++ parser_fix.cpp	2015-09-29 12:11:35.020180279 +0300
@@ -541,7 +541,7 @@
     break;
 
   case 6:
-    if (! (#line 46 "xml_parser/parser.ypp" // lalr1.cc:859
+    if (! (//#line 46 "xml_parser/parser.ypp" // lalr1.cc:859
      yystack_[5].value.as< std::string > () == yystack_[0].value.as< std::string > () )) YYERROR;
 #line 547 "xml_parser/parser.cpp" // lalr1.cc:859
     break;
